tests/conftest.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pytest
+import dataset
+from sqlalchemy import create_engine, MetaData
+from sqlalchemy.orm import sessionmaker
+import processors.base.config as config
+
+# Make fixtures available to all tests
+
+from tests.fixtures.files import file_fixture
+from tests.fixtures.trials import trial
+from tests.fixtures.sources import nct_source, fda_source
+from tests.fixtures.fda_approvals import fda_approval
+from tests.fixtures.fda_applications import fda_application
+from tests.fixtures.organizations import organization
+
+
+def teardown_database(engine):
+    metadata = MetaData(bind=engine)
+    metadata.reflect()
+    metadata.drop_all()
+
+
+def create_test_database(source_database_url, test_database_url):
+    engine = create_engine(source_database_url)
+    test_engine = create_engine(test_database_url)
+    metadata = MetaData()
+    metadata.reflect(engine)
+    metadata.create_all(test_engine)
+    return test_engine
```

Comment:
I think this is caused by the fact that it comes from a fork. I noticed env variables have been added to the `opentrials/processors` repository which I don't have in my fork. 

```
Setting environment variables from repository settings

$ export DOCKER_EMAIL=[secure]

$ export DOCKER_USER=[secure]

$ export DOCKER_PASS=[secure]

$ export WAREHOUSE_URL=[secure]

$ export DATABASE_URL=[secure]

```
Travis fails with the error:

```
  File "/home/travis/build/opentrials/processors/tests/conftest.py", line 11, in <module>

    import processors.base.config as config

  File "/home/travis/build/opentrials/processors/processors/base/config.py", line 16, in <module>

    WAREHOUSE_URL = os.environ['WAREHOUSE_URL']

  File "/home/travis/virtualenv/python2.7.10/lib/python2.7/UserDict.py", line 23, in __getitem__

    raise KeyError(key)

KeyError: u'WAREHOUSE_URL'
```
I'll make a PR from a branch here after I'm done with the functionality.